Add all new Models (opus, sonnet and haiku)

SarahTrees opened the pull request:
All new models (model aliases) have been added.
https://docs.claude.com/en/docs/about-claude/models/overview#model-aliases

The default model is now: claude-sonnet-4-5.

Old models have been retained, as some of them still work.

The version in the manifest has been changed from 1.2.1 to 1.3.1.

Thank you for checking!
Best regards!

iamgodot reviewed (changes requested):
Good, but still help me fix 2 things:

1. Remove the old models in the code, so just leave the 6 new models you added
2. Add a new line in versions.json for the new version, it'll mean the minimum app version for the plugin version, we can still use "0.15.0"

(you might want to amend your commits and force push instead of recreating them)

SarahTrees commented:
Thank you!

### Old Models
I checked  Model status  https://docs.claude.com/en/docs/about-claude/model-deprecations#model-status

I have removed all models that are no longer supported by Antropic.

For the old models that are still supported today, I have noted the status (Active/Deprecated) and also the ‘Tentative Retirement Date’.

This ensures that no models that no longer work are available. At the same time, however, you can still select models that still work today.

I myself sometimes use old models for specific tasks because I know that I will get the expected results (new is not always better...).

 
### Change tracking Model Status
I have added a task in change tracking: If the status of the Antropic models changes, I will adjust the plugin. (I also have to adjust such things in my tools).


###  amend  and force push instead of recreating
Thank you for the idea, I'll give that a try. I see that it seems my new changes are already visible here.
